core/src/main/java/com/linecorp/armeria/internal/http/Http1ObjectEncoder.java

```diff
@@ -79,7 +79,7 @@
     /**
      * The map which maps a request ID to its related pending response.
      */
-    private final IntObjectMap<PendingWrites> pendingWrites = new IntObjectHashMap<>();
+    private final IntObjectMap<PendingWrites> pendingWritesMap = new IntObjectHashMap<>();
```

Comment:
Renamed to avoid the confusion with a variable `pendingWrites` whose type is `PendingWrites`.